### DISCUSSION TITLE
GPS problem

### DISCUSSION THREAD

**Discussion by pimol** [mikaelnousiainen/RS41ng#36] [Q&A] [answered]
I load the .elf compiled and only the APRS is enabled in config.h
APRS messages are sent as expected but the coordinates are always 0000. Do I also need to enable GPS serial transmission? Or is there some other flag to set to enable GPS reading?
I tried with two different RS41, so I'm pretty sure it is a FW problem.

Pietro

**Comment by iz2fly** [comment]
Hello Pietro.
What frequency did you use?
I tried packet transmission at 432.500MHz with Si4032. I put the RS41 outside the house.
I transmitted 1000 frames, after a few seconds the decoding started, almost all of them were decrypted (85%), the GPS also worked almost all the time, I know the original program had some problems with the packet mode, I don't know if it has been improved here.
On my amateur radio site you find a better version of this mode of transmission made by my friend.
I also tried the 144 MHz WSPR mode and it works quite well stabilizing the frequency with an OCXO.
Good hunting.

**Reply by pimol** [reply]
The same, Michael. 432.5 and I decode the messages in local with soundmodem; the string of position was all to 0. 
Today I put the RS41 outside, open air and it works! My shame. Thank you

**Reply by pimol** [reply, marked as answer]
Actually, was flagged the 3D, too. With 2D position the RS41 locks also inside.

73 I2OIM

**Comment by iz2fly** [comment]
@pimol 
No problem, it happened to me for other things.
Ernesto

**Comment by mikaelnousiainen** [comment]
Thanks for the discussion. It seems the problem was solved: the RS41 radiosondes indeed need a clear view to the sky to obtain GPS lock. It does not work indoors at all. In my experience, the GPS reception gets disturbed very easily by nearby objects and obstructions.

73,
Mikael OH3BHX
